test/instrumentation/main.go

```diff
@@ -173,3 +179,100 @@ func globalVariableDeclarations(tree *ast.File) map[string]ast.Expr {
 	}
 	return consts
 }
+
+func importedGlobalVariableDeclaration(localVariables map[string]ast.Expr, imports []*ast.ImportSpec) (map[string]ast.Expr, error) {
+	// to test the import will be rooted at GOPATH, so probably import will be something like /test/instrumentation/testpackage
+
+	// env gets
+	//GOMODCACHE := os.Getenv("GOMODCACHE")
+	GOROOT := os.Getenv("GOROOT")
+	GOOS := os.Getenv("GOOS")
+	KUBE_ROOT := os.Getenv("KUBE_ROOT")
+	KUBE_URL_ROOT := "k8s.io/kubernetes"
+	KUBE_ROOT_INTERNAL := strings.Replace(KUBE_ROOT, KUBE_URL_ROOT, "", 1) // k8s/k8s refs need this stripped
```

Comment:
these can be declared in outer scope no?

Reply:
yeah, make 'em top level `consts` even

Reply:
moved to top level `const` or `var` as appropriate.  I'm open to suggestions on if there is a better way to pull env variables into this application.  I would normally have a config package or something but I'm new to executing code via `go run` like we are here.